meetups/2020/jan/25/participant_pitch.md

```diff
@@ -0,0 +1,30 @@
+Hello there, fellow developers!
+
+Greetings for the day. We hope you had a great time during this winter break
+and are all charged up cause we definitely are.  So, with a feeling of great
+enthusiasm and pride, we present to you Amazon Alexa powered “CODE-N-BUILD: Who
+Let The Codes Out” in association with DSC-JECRC.
+
+After having a hands-on experience with multiple successful local hackathons,
+we take the next step forward with the biggest hackathon JECRC University has
+ever witnessed. Code-N-Build aims to bring a widely known hackathon culture
+with our own added fun elements. The event spans over a duration of 30 hours
+brimming with coding, developing and not to miss fun activities.
+
+Why Code-N-Build?
+
+- Prizes worth up to Rs 2 lakhs.
+- Free registrations, food, and refreshments for all the participants.
+- Exciting goodies.
+- Communication and networking opportunities.
+- A healthy competitive environment with exposure and interaction opportunities with the professionals.
+- Opportunity to deal with real problems.
+- Fun activities to relax and reboot.
+
+and many more.
+
+Sounds exciting!! Doesn't it? You know what will be more exciting? You all being a part of what we believe is going to be an amazing experience!
+
+To Register Yourself and for more details visit our website [codenbuild.tech](https://codenbuild.tech/).
```

Comment:
Shouldn't the rules for team building be in this message if this one is the participant message?